isamples_metadata/SmithsonianTransformer.py

```diff
@@ -133,18 +139,18 @@ def has_context_categories(self) -> typing.List[dict[str, str]]:
                 self.source_record.get("higherClassification", ""),
             ]
         )
-        return [vocabulary_mapper.SAMPLED_FEATURE.term_for_label(category).metadata_dict() for category in categories]
+        return [vocabulary_mapper.sampled_feature_type().term_for_label(category)]
```

Comment:
This line was the source of the odd "a", "b", "c", "d", "e" we were seeing in the index for the Smithsonian records' controlled vocabularies.  When we converted things over to use the modelserver endpoint, the client code thought it was receiving a `list` but the modelserver is actually returning a string.  I've updated the API to reflect this.